src/main/fc/config.c

```diff
@@ -124,17 +124,12 @@ PG_RESET_TEMPLATE(pilotConfig_t, pilotConfig,
     .name = { 0 }
 );
 
-PG_REGISTER_WITH_RESET_TEMPLATE(systemConfig_t, systemConfig, PG_SYSTEM_CONFIG, 1);
+PG_REGISTER_WITH_RESET_TEMPLATE(systemConfig_t, systemConfig, PG_SYSTEM_CONFIG, 2);
 
-#ifndef USE_OSD_SLAVE
-#if defined(STM32F4) && !defined(DISABLE_OVERCLOCK)
+#ifdef USE_OSD_SLAVE
```

Comment:
To be honest, I think we should also do away with the special case for `OSD_SLAVE`. Parameter groups do have a group level granularity, and everything below that should be handled by particular targets that do not support functionality just ignoring this functionality. Once we are introducing 'PG over MSP' and most likely generated stubs for it, the current code will force us to keep track not only of SYSTEM_CONFIG.1 and SYSTEM_CONFIG.2, but also of SYSTEM_CONFIG(osd_slave).1 and SYSTEM_CONFIG(osd_slave).2.

Reply:
I think you're right. However let's do that in a separate PR, since there are probably other PGs that are OSD_SLAVE dependent and we should fix them all in one PR.